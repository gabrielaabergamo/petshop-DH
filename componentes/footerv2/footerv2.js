export function createMarkupFooterv2(){
    return(`
    <p class="footer-copyright">PETSHOP DH -  2021</p>
    `)
}