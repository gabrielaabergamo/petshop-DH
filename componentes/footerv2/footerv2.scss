.footer-copyright{
    text-align: center;
    padding: 15px;
    margin-bottom: 0;
    background-color:$background-footer;
} 