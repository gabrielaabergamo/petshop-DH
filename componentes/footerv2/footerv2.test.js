import {createMarkupFooterv2} from './footerv2'

test("createMarkupFooterv2 should render header", () => {
    expect(createMarkupFooterv2()).toMatchSnapshot()
})