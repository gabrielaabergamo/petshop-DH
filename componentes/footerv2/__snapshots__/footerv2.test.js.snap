// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['createMarkupFooterv2 should render the footer\'s second version 1'] = `
"
    <p class=\\"footer-copyright\\">PETSHOP DH -  2022</p>
    "
`;
